File: kotlin/1929-e-concatenation-of-array.kt
// https://leetcode.com/problems/concatenation-of-array
class Solution {
    fun getConcatenation(nums: IntArray): IntArray {
        val n = nums.size
        val ans = IntArray(n * 2)

        System.arraycopy(nums, 0, ans, 0, n)
        System.arraycopy(nums, 0, ans, n, n)

        return ans
    }
}

File: python/1929-e-concatenation-of-array.py
## https://leetcode.com/problems/concatenation-of-array
from typing import List
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        return nums + nums

if __name__ == '__main__':
    print(Solution().getConcatenation([1,2,3]))